### PR TITLE
Updated dependencies including Husky

### PR DESCRIPTION
**Pull request by pozil**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,8 @@
         "test:unit:coverage": "sfdx-lwc-jest --coverage",
         "prettier": "prettier --write \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
         "prettier:verify": "prettier --list-different \"**/*.{cls,cmp,component,css,html,js,json,md,page,trigger,xml,yaml,yml}\"",
-        "postinstall": "node bin/check-version.js"
+        "postinstall": "node bin/check-version.js && husky install",
+        "precommit": "npm run prettier && npm run test"
```
Comment (albarivas) [inline package.json:17]: what about lint?

File: .husky/pre-commit [hunk 2]
```diff
@@ -0,0 +1 @@
+npm run precommit
```
Comment (albarivas) [inline .husky/pre-commit:1]: I created the hook following husky instructions (npx approach) and this is what was created - https://github.com/developerforce/lwc-builder/blob/main/.husky/pre-commit, not sure why yours is different?

Comment (albarivas) [reply .husky/pre-commit:1]: I've said nothing, it's an issue with yarn and windows so no worries - https://typicode.github.io/husky/#/?id=yarn-on-windows

File: package.json (hunk 1 above)
Comment (pozil) [reply package.json:17]: @albarivas if you look a couple of lines up on line 9, lint is included in `npm run test` (I know, it's weird, not my choice)

**Review by albarivas — APPROVED** [review]

**Pull request merged**